src/features/exercises/components/ReadyModal.tsx

```diff
@@ -1,3 +1,4 @@
+import { JSX } from 'react/jsx-runtime';
```

Comment:
Is it okay to have it like this? I think I have no preference and either way is ok.

As I noticed that it is also possible to use just:
```tsx
import { JSX } from 'react';
```